code/modules/events/space_dragon.dm

```diff
@@ -0,0 +1,45 @@
+/datum/round_event_control/space_dragon
+	name = "Spawn Space Dragon"
+	typepath = /datum/round_event/ghost_role/space_dragon
+	weight = 8
+	max_occurrences = 1
+	min_players = 20
```

Comment:
you're putting a lot of faith into a four hundred health antag that can permanently hide corpses, move freely in space, heal, and hardstun people plus summon a tide of carp for just twenty people on >citadel